### PR TITLE
fix: language pack code unification

### PR DESCRIPTION
**Pull request by SQLGate**
## Description

Added Korean language pack.
Also, we have organized the Simplified Chinese and English language packs consistently.

### CODE REVIEW

**Comment by wewoor** [comment]
@SQLGate Are you a Korean developer?   We are glad to see the PR to Molecule, we are going to merge after review.

File: src/extensions/locales-defaults/locales/en.json [hunk 1]
```diff
@@ -24,14 +26,18 @@
...
+        "menu.defaultProjectName": "Default Project Name",
         "menu.run": "Run",
+        "menu.runTask": "Run Task",
         "menu.help": "Help",
+        "menu.about": "About",
         "menu.menuBarHorizontal": "Menu Bar Horizontal Mode",
         "menu.menuBarVertical": "Menu Bar Vertical Mode",
-        "sidebar.explore.title": "Explorer",
         "sidebar.explore.folders": "Folders",
+        "sidebar.explore.title": "Explore",
```
Comment (wewoor) [inline src/extensions/locales-defaults/locales/en.json:40]:
```suggestion
        "sidebar.explore.title": "Explorer",
```
Keep the same abstraction with VS Code.

**Review by wewoor — CHANGES_REQUESTED** [review]

File: src/extensions/locales-defaults/locales/en.json (hunk 1 above)
Comment (SQLGate) [reply src/extensions/locales-defaults/locales/en.json:40]: Good!. My mistake.

**Comment by SQLGate** [comment]
> @SQLGate Are you a Korean developer? We are glad to see the PR to Molecule, we are going to merge after review.

Yes! I am Korean developer and Delphi developer. This project is the one I've been looking for.
I'm making a database solution called SQLGate
http://www.sqlgate.com
and trying to get this product to work on the web.
So I was looking for an IDE JavaScript framework and I was grateful to see this source.

**Review by wewoor — APPROVED** [review]
+1

**Comment by wewoor** [comment]
@SQLGate Unit tests failed, there need you to update the snapshots.

**Comment by SQLGate** [comment]
> @SQLGate Unit tests failed, there need you to update the snapshots.
I will Unit tests, Again check code

**Comment by wewoor** [comment]
> > @SQLGate Unit tests failed, there need you to update the snapshots.
> > I will Unit tests, Again check code

If you have any questions,  we also can communicate on Discord: https://discord.com/invite/b62gpHwNA7

**Comment by SQLGate** [comment]
> 

How can i unit test it?

**Comment by wewoor** [comment]
> > 
> 
> How can i unit test it?

At the root folder, run `yarn test` command

**Comment by wewoor** [comment]
@SQLGate Maybe we need to extract the locales into an isolated package in the future.

**Pull request merged**